src/core/Akka/Util/Base64Encoding.cs

```diff
@@ -24,14 +25,38 @@ public static class Base64Encoding
         /// </summary>
         /// <param name="value">TBD</param>
         /// <returns>TBD</returns>
-        public static string Base64Encode(this long value) => Base64Encode(value, new StringBuilder()).ToString();
+        public static string Base64Encode(this long value)
+        {
+            return Base64Encode(value, "");
+        }
+
+        public static string Base64Encode(this long value, string prefix)
+        {
+            Span<char> sb = stackalloc char[11 + prefix?.Length ?? 0];
```

Comment:
Testing program is here:

https://gist.github.com/Aaronontheweb/449d565754d5f9bce475653842317277

`11` is the number of characters it takes to represent `long.MaxValue` in base64 encoded strings